src/containers/AlbumsView.jsx

```diff
@@ -32,17 +32,18 @@ export class AlbumsView extends Component {
       return <ErrorComponent errorType='albums' />
     }
     return (
-      <div>
-        <Topbar viewName='albums' />
-        <AlbumsList {...this.props} {...this.state} />
-        { this.props.children }
-      </div>
+      !this.props.children
```

Comment:
Would you mind explaining what's going on here? I can't find where this component is used with a `children`prop.